lessons/pydata/scikitlearn_api/index.ipynb

```diff
@@ -0,0 +1,1207 @@
+{
```

Comment:
<p>Pokud to dále nepoužijeme jako sloupec v Dataframe, spokojil bych se s něčím jako print(predikce_platu[:10]) místo převodu na Series jen pro použití metody head()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>